### ISSUE TITLE
How can I test my model on MPII test set?

### ISSUE THREAD

**Issue by xtyDoge** [leoxiaobin/deep-high-resolution-net.pytorch#84]
There are fields like `joints_vis` and `joints` in train.json or valid.json for MPII datasets, but I cannot find them in test.json? 
So how can I test my model on test set and have an output like Table `Performance comparisons on the MPII test set` in your paper? THX

**Comment by ustcjinggg** [comment]
Have you solved your problem, I have the same question
